container/package.json

```diff
@@ -48,11 +49,10 @@
     "eslint-plugin-svelte": "^2.33.0",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
-    "rollup": "2.78.0",
-    "rollup-plugin-copy": "^3.4.0",
+    "rollup": "4.22.5",
```

Comment:
The bundle process on **container** seems to fail with the error shown in the end.

I am aware that we can somehow delete **npm_modules** folder and try again, but I think that is a bit invasive, since there is still an[ open issue](https://github.com/npm/cli/issues/4828) that is not resolved around this, so the current solution is more of a workaround . Since the container build is quite critical, I would propose to instead update to closest patch to v2, that fixes the warning mentioned and doesnt affect build process, and has no workarounds.

 We can than think later about migrating to a later version when they fix their own issues. 

```
Error when trying to bundle packages in container 
Stopping execution of the process due to error: Error: Command failed: cd container && npm run bundle
 /luigi/container/node_modules/rollup/dist/native.js:59
		throw new Error(
		^

Error: Cannot find module @rollup/rollup-darwin-x64. npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). Please try `npm i` again after removing both package-lock.json and node_modules directory.
```